tock/tock/templates/projects/project_detail.html

```diff
@@ -18,4 +19,35 @@ <h3 class="chart__title">User Hours Over Time</h3>
   </utilization-chart>
 </figure>
 
+<table id="dataTable" class="striped">
+  <thead>
+    <th>Name</th>
+    {% for period in table.periods %}
+      <th>{{ period }}</th>
+    {% endfor %}
+  </thead>
+  <tbody>
+    {% for user, rows in table.groups.items %}
+    <tr>
+      <td>{{ user }}</td>
+      {% for period in table.periods %}
+      <td>{{ rows|get:period|default:0.0 }}</td>
+      {% endfor %}
+    </tr>
+    {% endfor %}
+  </tbody>
+</table>
+
+<link rel="stylesheet" href="//cdn.datatables.net/1.10.7/css/jquery.dataTables.min.css" />
+<script src="//cdn.datatables.net/1.10.7/js/jquery.dataTables.min.js"></script>
+<script type="text/javascript">
+$(document).ready(function() {
```

Comment:
How long will this table get? Could the paging be turned on after 3 or 4 reporting periods?